core/views.py

```diff
@@ -1,10 +1,64 @@
-from rest_framework import permissions
-from rest_framework import generics
-from rest_framework import status
-from .serializers import HackathonSerializer
+from rest_framework import generics, status, permissions
+from rest_framework.response import Response
+from django.utils import timezone
+import pytz
```

Comment:
Why this?

Reply:
Forgot to remove pytz, it's not required